app/templates/buy-checkout.jade

```diff
@@ -0,0 +1,72 @@
+form.bc-form(
+  name="checkoutForm"
+  ng-submit="enabled ? buy() : enableBuy()"
+  novalidate
+  autocomplete="off")
+  fieldset(ng-disabled="locked")
+    .group.mts
+      .item
+        .flex-between.flex-baseline.mbs
+          span.f-16.em-500.colon Enter Amount
+          span
+            div.flex-row.flex-center(ng-show="state.rate && !state.loadFailed")
+              span 1 {{ bitcoin.code }} = ${{ state.rate }}
+              helper-button(content="EXCHANGE_RATE_HELPER")
+            .state-danger-text(ng-show="state.loadFailed")
+              i.ti-alert
+              span.mls Failed to load exchange data
+        .flex-center
+          .flex-1.flex-center.border
+            input.form-control.no-border(
```

Comment:
Do you think it would make sense to add a focus-when here? Might be tricky because it will effect both sfox/checkout and onStep('buy')